dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/PoolManager.java

```diff
@@ -2367,6 +2370,196 @@ public void purgePoolContents() throws ResourceException {
         }
     }
 
+    public String abortConnections(String actionName, Object[] params) throws ResourceException, MBeanException {
+        boolean debugEnabled = TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled();
+        boolean entryExitEnabled = TraceComponent.isAnyTracingEnabled() && tc.isEntryEnabled();
+
+        if (entryExitEnabled) {
+            Tr.entry(tc, "abortConnections");
+        }
+
+        long maxInUseIdleValue = 0;
+        if (params[1] instanceof String) {
+            maxInUseIdleValue = Long.parseLong((String) params[1]);
+        } else if (params[1] instanceof Long) {
+            maxInUseIdleValue = ((Long) params[1]).longValue();
+        } else if (params[1] instanceof Integer) {
+            maxInUseIdleValue = ((Integer) params[1]).longValue();
+        } else {
+            Tr.error(tc, "INVALID_MBEAN_INVOKE_PARAM_J2CA8060", Arrays.toString(params), actionName);
+            String translatedError = Tr.formatMessage(tc, "INVALID_MBEAN_INVOKE_PARAM_J2CA8060", Arrays.toString(params), actionName);
+            throw new MBeanException(new IllegalArgumentException(translatedError));
+
+        }
+
+        String errorString = null;
+        StringBuffer sb = new StringBuffer();
+        updateToPoolInProgress = true;
+        synchronized (updateToPoolInProgressLockObject) {
+
+            long inuseAbortedHighValue = 0;
+            long inuseAbortedLowValue = Long.MAX_VALUE;
+            long inuseAbortedAverage = 0;
+            long inuseNotAbortedHighValue = 0;
+            long inuseNotAbortedLowValue = Long.MAX_VALUE;
+            long inuseNotAbortedAverage = 0;
+            int inuseNotAborted = 0;
+            boolean parkedConnectionExists = false;
+
+            try {
+                if (checkForActiveConnections(1)) {
+                    try {
+                        if (debugEnabled) {
+                            Tr.debug(tc, "Active connection pool activity defected, try again in 100 milliseconds");
```

Comment:
I think you mean "detected" instead of "defected"

Reply:
done